tls/s2n_client_extensions.c

```diff
@@ -198,12 +198,12 @@ int s2n_client_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
     }
 
     /*
-     * RFC 4492: Clients SHOULD send both the Supported Elliptic Curves Extension
-     * and the Supported Point Formats Extension.
+     * RFC 4492: Clients SHOULD send both the Supported Elliptic Curves Extension (renamed
+     * Supported Groups) and the Supported Point Formats Extension.
```

Comment:
perhaps mention that it is renamed in the tls 1.3 rfc to be more specific?

Reply:
i can update that